### DISCUSSION TITLE
How to change schema name on Swagger UI

### DISCUSSION THREAD

**Discussion by justinleahy** [poem-web/poem#681] [Q&A]
Is there a way to alter the name that the schema shows on the bottom of the swagger ui webpage?

![Web capture_19-10-2023_93612_localhost](https://github.com/poem-web/poem/assets/31375835/0201d889-ce2d-451e-a418-45bf4ad5795a)

For example, in the screenshot above it shows Model as the schema's name. Is there a way to alter this to User without changing the enums name?

Thank you very much,

Justin Leahy

**Comment by yudjinn** [comment]
this isnt an answer to renaming; but why are you not just naming it "User" in your codebase as well?
